qt_dotgraph/setup.py

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env python
 
 from distutils.core import setup
+
 from catkin_pkg.python_setup import generate_distutils_setup
 
 d = generate_distutils_setup(
     packages=['qt_dotgraph'],
-    package_dir={'': 'src'}
+    package_dir={'': 'qt_dotgraph'}
```

Comment:
This change doesn't seem the be the case - the Python package is still under `src` (and keeping it there sounds good)?

Same below.

Reply:
Yup. Changing back to src.